config/config.go

```diff
@@ -25,9 +25,9 @@ type config struct {
 func Load(l *logger.Logger) {
 	l = l.Create("CONFIG")
 	defer l.ChangeLevel(logger.LevelMain).Println("LOADED")
-	b, err := ioutil.ReadFile("config.json")
+	b, err := os.ReadFile("config.json")
```

Comment:
What's the point of changing it from ioutils to os here? It was still working fine, right?

Reply:
It will be removed in the future versions chk warnings in the editor.